### PR TITLE
Header Elements Laid Out and Positioned

### PR DESCRIPTION
**Pull request by thaitwo**

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
 					if ( is_front_page() && is_home() ) : ?>
 						<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></h1>
 					<?php else : ?>
-						<p class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></p>
+						<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></h1></p>
```
Comment (alexwolfe) [inline header.php:34]: 👍

File: sass/forms/_buttons.scss [hunk 2]
```diff
@@ -39,6 +39,7 @@ input[type="submit"] {
 	&.button-primary {
 		color: $color_white;
 		border: 1px solid $color_white;
+		margin-top: 25px;
```
Comment (alexwolfe) [inline sass/forms/_buttons.scss:42]: Looks like in general you're using tabs not spaces, I always recommend spaces :)

File: sass/layout/_front-layout.scss [hunk 3]
```diff
@@ -33,15 +20,15 @@
...
 	color: #263238;
 	text-align: center;
-	line-height: 60px;
 }
 
 .section-description {
+	font-family: "Raleway", sans-serif;
 	text-align: center;
-	padding: 0 30px;
+	padding: 5px 30px;
```
Comment (alexwolfe) [inline sass/layout/_front-layout.scss]: I recommend coming up with a `$unit` var in sass that you can use as the base for your measurements so throughout your design things are consistent. ex: 

```
$unit: 5px; 

 .section-description {
  padding: $unit ($unit * 6); 
 ... 
```

File: sass/layout/_layouts.scss [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+	content: "";
+	display: table;
+	clear: both;
+}
+
+.float-l {
+	float: left;
+}
+
+.float-r {
```
Comment (alexwolfe) [inline sass/layout/_layouts.scss]: .l-left and .l-right would probably make more sense since you are prefixing layout items with l-

**Review by alexwolfe — COMMENTED** [review]
Nice works 👍  Just a few comments/tips.

**Comment by thaitwo** [comment]
Thanks for the tips! I just revised some of the mistakes along with a few others non related to the header.
When you have a chance, can you take a look over again and merge if it looks good? Thanks 🍔

**Comment by thaitwo** [comment]
Yikes...sorry for all those file name changes!

**Pull request merged**